pkg/server/serverpb/status.proto

```diff
@@ -382,6 +382,18 @@ message RaftState {
   util.hlc.Timestamp lead_support_until = 8 [(gogoproto.nullable) = false];
 }
 
+// RACState contains the state of the Replication Admission Control component of
+// a range.
+message RACState {
```

Comment:
What should we include here? I'm thinking both the `RangeController` state (such as `nextRaftIndex`, `forceFlushIndex`), and per-stream (tokens held, `indexToSend`).

Reply:
Done
<!-- Reviewable comment -OEU9IBo2kS21ilZ6_FN:bb6un1j -->
<!-- Sent from Reviewable.io -->